packages/deployer/subtasks/generate-router.js

```diff
@@ -36,7 +34,12 @@ subtask(
 
   const binaryData = _buildBinaryData({ selectors });
 
-  const packageJson = readPackageJson();
+  let packageJson;
+  try {
+    packageJson = readPackageJson();
+  } catch (err) {
+    packageJson = { name: '' };
+  }
```

Comment:
This is so that the root project doesn't require the user to have a `package.json`